addon/index.js

```diff
@@ -69,7 +70,7 @@ export function unbindKeyboardShortcuts(context) {
     }
     object.detachEvent('on' + type, callback);
   };
-  context._mousetraps.forEach(mousetrap => {
+  A(context._mousetraps).forEach(mousetrap => {
```

Comment:
Instead of using Ember Array, could you check if the value is an array?

Reply:
Gotcha. How do you like it now?